packages/hooks/use-aria-multiselect/src/use-multiselect-state.ts

```diff
@@ -76,6 +76,7 @@ export function useMultiSelectState<T extends {}>(props: MultiSelectProps<T>): M
 
   const validationState = useFormValidationState({
     ...props,
+    // TODO: Future enhancement to support "aria" validation behavior.
```

Comment:
Select uses a custom implementation to support multiple selections, so adding support for `validationBehavior` might take some time.